manifests/g/Gilgamech/MoarShopBot/1.1552/Gilgamech.MoarShopBot.installer.yaml

```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.10.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.10.0.schema.json
+
+PackageIdentifier: Gilgamech.MoarShopBot
+PackageVersion: "1.1552"
+InstallerType: portable
+Commands:
+- shopbot
+Installers:
+- Architecture: x64
+  InstallerUrl: https://raw.githubusercontent.com/Gilgamech/MoarShopBot/refs/heads/main/MoarShopBot.exe

```

Comment:
This isn’t a versioned URL :(

It would be best to put the binary into a GitHub Release rather than having it as a file in a branch